minorminer/utils/raster_embedding.py

```diff
@@ -306,24 +327,36 @@ def raster_embedding_search(S, T, timeout=10, raster_breadth=None,
         raise ValueError("source graphs must a graph constructed by "
                          "dwave_networkx as chimera, pegasus or zephyr type")
 
-    _T = T.copy()
     embs = []
-    for i, f in enumerate(sublattice_mappings(tile, _T)):
-        Tr = _T.subgraph([f(n) for n in tile])
-
-        sub_embs = find_multiple_embeddings(
-            S, Tr,
-            max_num_emb=max_num_emb,
-            timeout=timeout)
-        embs += sub_embs
-        if len(embs) >= max_num_emb:
+    if inplace:
+        _T = T
```

Comment:
Similar to above. I would prefer to see lines 333-341 deleted and lines 344-359 unindented